src/index.html

```diff
@@ -5,9 +5,68 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <h1></h1>
+    <div
+      class="product">
+      <div
+        class="product-card"
+        data-qa="card"
+        >
+        <div class="product-image-container">
+          <img
+              class="product-image"
+              src="../src/images/imac.jpeg"
+              alt="image"
+            >
```

Comment:
Should be only 2 spaces, need to use more descriptive name for alt
```suggestion
          <img
            class="product-image"
            src="../src/images/imac.jpeg"
            alt="image"
          >
```